src/main/java/seedu/dietmanager/DailyFoodRecord.java

```diff
@@ -100,16 +101,114 @@ public String showLunch() {
     public String showDinner() {
         String message = "";
         for (Food food : dinner) {
-            message = message + food.getFoodName() + ", ";
+            message = message + food.toString();
         }
         if (message.equals("")) {
-            message = "Oops, you have no record for this meal";
-            return message;
+            message = "Oops, you have no record for this meal.\n";
```

Comment:
magic string

Reply:
again, not written by me, but i can shift it to bank